index.src.html

```diff
@@ -774,37 +778,36 @@ <h4 id="fetching">Fetching</h4>
           [=request/policy container=] accordingly, or to directly set
           |request|'s [=request/policy container=] to a non-null value.
 
-      1.  Let |clientAddressSpace| be |request|'s [=request/policy container=]'s
-          [=policy container/IP address space=]
-
-      1.  If |request|'s [=request/client=] is not a [=secure context=]
-          (including if it is null), and |connection|'s [=connection/IP address
-          space=] is [=IP address space/less public=] than |clientAddressSpace|,
-          then return a [=network error=].
-
       1.  If |request|'s [=request/target IP address space=] is not null, then:
 
+          1. [=Assert=]: |request|'s [=request/target IP address space=] is
```

Comment:
I think we can replace this assert and fix #109: we can instead assert that the target IP address space is not public (or we could say "is less public than public", but that seems more confusing).